Block/Checkout/Minicart.php

```diff
@@ -126,7 +126,8 @@ public function isActive()
     {
         return $this->payment->isAvailable($this->checkoutSession->getQuote()) &&
             $this->helperData->allowMinicartCheckout() &&
-            !$this->isCartView();
+            !$this->isCartView() &&
+            $this->quoteHelper->aboveThreshold($this->quoteHelper->getGrandTotal()/100);
```

Comment:
I liked when you broke this out into a variable in a couple of other spots, so I think doing that here also would be good

Reply:
Changed